data_analysis/price_prediction.py

```diff
@@ -6,25 +6,34 @@
 from datetime import datetime, timedelta
 import pandas as pd
 from statsmodels.tsa.arima_model import ARIMA
+#import sys
+#import os
+#sys.path.append(os.path.abspath('../get_data'))
+from get_data.importData import call_macro
+#from get_data.get_data import call_api
 
-from get_data.get_data import call_api
-
+file = open('./get_data/macrotrends_data.csv', 'r')
 
 def model_arima(n_days, n_predictions, current_value):
     """Returns an array of predicted prices for given number of days using
     Quandl API data."""
 
-    data = call_api(n_days)
+    data = call_macro(n_days, file)
+    #data = call_api(n_days)
     df = pd.DataFrame(data)
 
     # If last date in data frame is not current date append
     # data frame with current date and price
     current_time = datetime.now()
-    if current_time.date() > df.Date.iloc[-1].date():
-        df = df.append(
-            {"Date": current_time, "Value": current_value}, ignore_index=True
-            )
-    model = ARIMA(df[["Value"]], order=(1, 1, 1))
+    #current_time_string = current_time.strftime('%d/%m/%Y')
+    #current_time_formatted = datetime.strptime(current_time_string, '%d/%m/%Y')
+    #last_date_on_csv_formatted = datetime.strptime(df['date'].iloc[-1], '%d/%m/%Y')
+    #if current_time_formatted > last_date_on_csv_formatted:
+    #    df = df.append(
+    #        {"date": current_time, "value": current_value}, ignore_index=True
+    #        )
+    #print ("This is what the fuck df is: ", df[['date', 'value']])
+    model = ARIMA(df["value"], order=(1, 1, 1))
```

Comment:
Please can you remove all the commented out code, its fine to delete if you're happy your new code works :) also it's in git history if we ever want it back so its safe to delete code 👍

Reply:
LOL forgot about that comment!